kafka-client/src/main/java/dev/responsive/kafka/internal/db/mongo/MongoWindowedTable.java

```diff
@@ -43,6 +43,7 @@
 import dev.responsive.kafka.internal.db.partitioning.Segmenter.SegmentPartition;
 import dev.responsive.kafka.internal.db.partitioning.WindowSegmentPartitioner;
 import dev.responsive.kafka.internal.stores.RemoteWriteResult;
+import dev.responsive.kafka.internal.utils.DuplicateKeyListValueIterator;
```

Comment:
`MongoWindowedTable::fetch` for point lookups returns null for duplicate stores - is that what we want? Or should we have the same behaviour as CommitBuffer (return the first value)

Reply:
Ah, good catch, no we should just return the first value (though arguably it's not really a defined API for duplicate stores and I would not recommend anyone use it. Of course I'm sure some people still do anyways 🤷‍♀️ )